api/models.py

```diff
@@ -1,3 +1,22 @@
 from django.db import models
-
+from django.utils import timezone
 # Create your models here.
+
+class Task (models.Model):
+
+	STATUS_CHOICES = (
+	('UNASSIGNED', 'Unassigned'),
+	('WORKINGON', 'Working On'),
+	   ('COMPLETED', 'Completed'),
```

Comment:
did you actually run anything with this code? you have mixed indentation characters, which python isn't too happy about